DigitalOcean.API/Models/Requests/Image.cs

```diff
@@ -1,18 +1,47 @@
-﻿using Newtonsoft.Json;
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
 
 namespace DigitalOcean.API.Models.Requests {
-    public enum ImageType {
-        All,
-        Application,
-        Distribution,
-        Private
-    }
-
     public class Image {
         /// <summary>
-        /// The new name that you would like to use for the image.
+        /// The display name to be given to an image.
```

Comment:
Class was previously used in `Update` call, this class now operates on all properties necessary for `Create` call.